soak/autoscaler/init.go

```diff
@@ -0,0 +1,63 @@
+package autoscaler
+
+import (
+	"fmt"
+	"github.com/apache/yunikorn-release/soak/framework"
+	"github.com/apache/yunikorn-release/soak/logger"
+	"go.uber.org/zap"
+	"os/exec"
+	"strings"
+)
+
+type AutoscalingScenario struct {
+	templateConf framework.Template
+	testCases    []framework.TestCase
+}
+
+func New(config *framework.Config) *AutoscalingScenario {
+	for _, c := range config.Tests {
+		if c.Name == "autoscaling" {
+			return &AutoscalingScenario{
+				templateConf: c.Template,
+				testCases:    c.TestCases,
+			}
+		}
+	}
+	return nil
+}
+
+func (a *AutoscalingScenario) GetName() string {
+	return "autoscaling"
+}
+
+func (a *AutoscalingScenario) Init() error {
+	return nil
+}
+
+func (a *AutoscalingScenario) Tests() []framework.TestCase {
+	// enable or disable test cases here
+	return a.testCases
+}
+
+func (a *AutoscalingScenario) Run() ([]string, error) {
+	log := logger.Logger
+	results := make([]string, len(a.testCases))
+	for idx, tests := range a.testCases {
+		clusterLoaderConfigPath := tests.ClusterLoaderConfigPath
+		args := []string{fmt.Sprintf("----testconfig=%s", clusterLoaderConfigPath),
+			"--provider=kind", "--kubeconfig=/Users/b_achar/.kube/config",
```

Comment:
"--kubeconfig=/Users/b_achar/.kube/config"

is this supposed to be hard coded here?

Reply:
Fixed